### PR TITLE
Adding ability to get Marathon plugins

### PR DESCRIPTION
**Pull request by kensipe**

### CODE REVIEW

**Comment by kensipe** [comment]
rebased

**Comment by kensipe** [comment]
rebased

**Comment by kensipe** [comment]
rebased

**Comment by kensipe** [comment]
@klueska @bamarni is there anything preventing a merge?

**Comment by kensipe** [comment]
rebased

File: cli/dcoscli/data/help/marathon.txt [hunk 1]
```diff
@@ -28,6 +28,7 @@ Usage:
     dcos marathon group show [--group-version=<group-version>] <group-id>
     dcos marathon group remove [--force] <group-id>
     dcos marathon group update [--force] <group-id> [<properties>...]
+    dcos marathon plugins [--json]
```
Comment (bamarni) [inline cli/dcoscli/data/help/marathon.txt]: By analogy with the other marathon subcommands, where it is a singular noun followed by an action, should this rather be `dcos marathon plugin list [--json]`? That'd make it easier to add plulgin related subommands in the future while keeping backward compatibility.

Comment (kensipe) [reply cli/dcoscli/data/help/marathon.txt]: makes sense to me!

**Comment by kensipe** [comment]
run linux integration tests

**Comment by kensipe** [comment]
run linux integration tests

**Comment by kensipe** [comment]
the failure is unrelated to the change :(

**Comment by bamarni** [comment]
The failure seems related : 

```
[linux-tests] CMD: ['dcos', 'marathon', '--info']
[linux-tests] STDOUT: 
[linux-tests] STDERR: Traceback (most recent call last):
[linux-tests]   File "/workspace/dcos-cli/cli/.tox/py35-integration/lib/python3.5/site-packages/dcoscli/subcommand.py", line 103, in run_and_capture
[linux-tests]     exit_code = m.main([self._command] + self._args)
[linux-tests]   File "/workspace/dcos-cli/cli/.tox/py35-integration/lib/python3.5/site-packages/dcoscli/marathon/main.py", line 23, in main
[linux-tests]     return _main(argv)
[linux-tests]   File "/workspace/dcos-cli/cli/.tox/py35-integration/lib/python3.5/site-packages/dcoscli/util.py", line 24, in wrapper
[linux-tests]     result = func(*args, **kwargs)
[linux-tests]   File "/workspace/dcos-cli/cli/.tox/py35-integration/lib/python3.5/site-packages/dcoscli/marathon/main.py", line 36, in _main
[linux-tests]     return cmds.execute(_cmds(), args)
[linux-tests]   File "/workspace/dcos-cli/dcos/cmds.py", line 38, in execute
[linux-tests]     if not args[positional]:
[linux-tests] KeyError: 'plugins'
```

File: cli/dcoscli/data/help/marathon.txt [hunk 2]
```diff
@@ -28,6 +28,7 @@ Usage:
     dcos marathon group show [--group-version=<group-version>] <group-id>
     dcos marathon group remove [--force] <group-id>
     dcos marathon group update [--force] <group-id> [<properties>...]
+    dcos marathon plugin [--json]
```
Comment (bamarni) [inline cli/dcoscli/data/help/marathon.txt]: here I meant `dcos marathon plugin [--json]`

Comment (kensipe) [reply cli/dcoscli/data/help/marathon.txt]: yeah... that makes sense and is symmetrical

**Comment by kensipe** [comment]
@bamarni updated based on feedback and the last test error was caused by my last change which was a mismatch between help txt and the command support.   It is all resolved.   Ready to go after tests are green... thx!!

**Comment by kensipe** [comment]
run linux integration tests

**Comment by bamarni** [comment]
run windows integration tests

**Comment by bamarni** [comment]
run mac integration tests

**Comment by kensipe** [comment]
run mac integration tests

**Comment by bamarni** [comment]
It looks like mac and windows integration tests fail in an unrelated way, I'll check this out on Monday. They are however not a hard requirement to get this PR merged.

**Comment by kensipe** [comment]
@bamarni any word here?

**Comment by bamarni** [comment]
hum clicked on rebase instead of squash, @kensipe in the future could you please make sure your commits are squashed in a meaningful way? That'd help keeping the history clean.

**Pull request merged**